src/main/java/seedu/agendum/storage/XmlToDoListStorage.java

```diff
@@ -70,4 +71,10 @@ public void saveToDoList(ReadOnlyToDoList toDoList, String filePath) throws IOEx
     public void saveToDoList(ReadOnlyToDoList toDoList) throws IOException {
         saveToDoList(toDoList, filePath);
     }
+
+    @Override
+    public void setToDoListFilePath(String filePath) {
+        assert StringUtil.isValidFilePath(filePath);        
```

Comment:
nope not intending to delete. Was thinking, for example if u have a word doc, if you "save as", it doesn't delete the previous files